src/servers/trigger.ts

```diff
@@ -166,7 +165,7 @@ export const onInvoiceUpdate = async (invoice) => {
 }
 
 const publishCurrentPrice = async () => {
-  const usdPerSat = await getCurrentPrice()
+  const usdPerSat = await Prices.getCurrentPrice()
```

Comment:
Or these renamings required for the wrappers to work or are you just moving to a consistent style?

Reply:
in servers we need to use `Accounts, Admin, Lightning, Prices, Users, Wallets` from `@app` if we want to use telemetry wrapped functions.